app/models/list.rb

```diff
@@ -1,7 +1,9 @@
 class List < ApplicationRecord
+  acts_as_taggable
+
   has_and_belongs_to_many :homepages
   belongs_to :user
-
+ 
```

Comment:
This is pretty unnecessary. There's a Sublime setting for trimming trailing whitespace. I recommend turning it on.